gateway-service-impl/src/main/java/org/hypertrace/gateway/service/common/util/AttributeMetadataUtil.java

```diff
@@ -153,9 +151,7 @@ public static AttributeMetadata getAttributeMetadata(
 
   private static AttributeMetadata getAttributeMetadata(
       AttributeMetadataProvider attributeMetadataProvider,
-      RequestContext requestContext,
-      AttributeScope scope,
-      String key) {
+      RequestContext requestContext, String scope, String key) {
```

Comment:
mmh I prefer line by line continuation

Reply:
Hmm.. interestingly, java code conventions plugin doesn't seem to lean towards one approach. Do you want me to fix this?

Reply:
sure.

Reply:
Done.